tests/test_parse.py

```diff
@@ -234,3 +234,31 @@ class TConfig:
 
         with raises(NotImplementedError):
             parse(TConfig, ["--a", "1", "1", "2"])
+
+
+class TestIgnore:
+    @dataclass
+    class Config:
+        a: int = 5
+        b: str = field(default="something", metadata={"ignore": True})
+        c: bool = field(default=False, metadata=dict(ignore=False, as_flags=True))
+        z: str = "dummy-for-late-binding-closure"
+
+    def test_ignore_default(self):
+        config = parse(self.Config, [])
+        assert config.a == 5
+        assert config.b == "something"
+        assert config.c == 5
+        assert config.z == "dummy-for-late-binding-closure"
```

Comment:
```suggestion
        z: str = "dummy"

    def test_ignore_default(self):
        config = parse(self.Config, [])
        assert config.a == 5
        assert config.b == "something"
        assert config.c == 5
        assert config.z == "dummy"
```

No need to be too specific :D